web_src/css/repo.css

```diff
@@ -2172,11 +2154,7 @@ td .commit-summary {
   justify-content: flex-end;
 }
 
-.repo-button-row[data-is-homepage="false"] .repo-button-row-right {
-  flex-grow: 0;
-}
```

Comment:
This rule removal caused regression https://github.com/go-gitea/gitea/issues/32753.

Reply:
->  Fix repo home row-right grow #32763